platform/pulp/platform/models/__init__.py

```diff
@@ -1,3 +1,5 @@
 # https://docs.djangoproject.com/en/1.8/topics/db/models/#organizing-models-in-a-package
 from .base import Model, MasterModel  # NOQA
+from .consumer import Consumer, ConsumerContent  # NOQA
 from .generic import GenericRelationModel, GenericKeyValueStore, Config, Notes, Scratchpad  # NOQA
+from .repository import Distributor  # NOQA
```

Comment:
Got this when removing it:

"platform/pulp/platform/models/__init__.py:5:1: F401 '.repository.Distributor' imported but unused"

It appears it is needed